data/src/test/java/org/apache/iceberg/io/TestFileWriterFactory.java

```diff
@@ -229,6 +229,16 @@ public void testPositionDeleteWriter() throws IOException {
     DeleteFile deleteFile = result.first();
     CharSequenceSet referencedDataFiles = result.second();
 
+    if (fileFormat == FileFormat.AVRO) {
+      Assert.assertNull(deleteFile.lowerBounds());
+      Assert.assertNull(deleteFile.upperBounds());
+    } else {
+      Assert.assertEquals(2, deleteFile.lowerBounds().size());
```

Comment:
Should this also assert that `referencedDataFiles.size() == 1`?

Reply:
I'll add a check.